.travis.yml

```diff
@@ -2,6 +2,10 @@ language: php
 
 dist: trusty
 
+sudo: required
```

Comment:
How come this has been added? What are we using `sudo` for?

Reply:
ok, I've removed this and travis is green

Reply:
I made many changes until things stopped breaking... this was one of those changes that happened to make the tests pass. :)